leobot_web_server/web_content/vr.html

```diff
@@ -4,6 +4,7 @@
         <meta charset="utf-8">
         <title>Leobot</title>
         <link rel="stylesheet" type="text/css" href="css/vr.css" />
+        <script src="https://static.robotwebtools.org/roslibjs/current/roslib.min.js"></script>
```

Comment:
please use roslib.min.js from nodejs package. Discuss with @MaxZhaloba how to do it :)